crown-court-proceeding/src/test/java/uk/gov/justice/laa/crime/crowncourt/config/CrownCourtProceedingTestConfiguration.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.justice.laa.crime.crowncourt.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.security.oauth2.core.AuthorizationGrantType;
+import org.springframework.security.oauth2.core.ClientAuthenticationMethod;
+import org.springframework.security.oauth2.server.authorization.client.JdbcRegisteredClientRepository;
+import org.springframework.security.oauth2.server.authorization.client.RegisteredClient;
+import org.springframework.security.oauth2.server.authorization.client.RegisteredClientRepository;
+import org.springframework.security.oauth2.server.authorization.config.ClientSettings;
+
+import java.util.UUID;
+
+@Configuration(proxyBeanMethods = false)
+public class CrownCourtProceedingTestConfiguration {
```

Comment:
Do we need this right now? The authorisation server config hasn't been added yet

Reply:
Authorization is now added.